openstack/workflow/v2/executions/results.go

```diff
@@ -0,0 +1,83 @@
+package executions
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// CreateResult is the response of a Post operations. Call its Extract method to interpret it as an Execution.
+type CreateResult struct {
+	commonResult
+}
+
+// Extract helps to get an Execution struct from a Get or a Create function.
+func (r commonResult) Extract() (*Execution, error) {
+	var s Execution
+	err := r.ExtractInto(&s)
+	return &s, err
+}
+
+// Execution represents a workflow execution on OpenStack mistral API.
+type Execution struct {
```

Comment:
It looks like this might be missing some fields. Can you confirm:

* https://github.com/openstack/mistral/blob/master/mistral/db/v2/sqlalchemy/models.py#L170
* https://github.com/openstack/mistral/blob/master/mistral/db/v2/sqlalchemy/models.py#L214

As well, can we confirm if the JSON fields have JSON embedded or are they sent back as actual data structures?

Reply:
Missing fields added! And some for cron triggers, I worked on JSON fields to be ```map[string]interface{}```